core/internal/corelib/summarylib.go

```diff
@@ -18,8 +19,13 @@ func JsonifyItems[V genericItem](items []V) (string, error) {
 	jsonMap := make(map[string]interface{})
 
 	for _, item := range items {
-		var value interface{}
-		if err := json.Unmarshal([]byte(item.GetValueJson()), &value); err != nil {
+		var (
+			value interface{}
+			err   error
+		)
+		// TODO: use simplejsonext for now until we replace the usage of json with protocol buffer
+		// and proto json marshaler
+		if value, err = json.Unmarshal([]byte(item.GetValueJson())); err != nil {
```

Comment:
oh thanks, forgot to change it, was doing it that way so it would be easier for me to go back and forth between the segmentio and kent's implementation